### DISCUSSION TITLE
Support for Laravel 5.8 / 6.0

### DISCUSSION THREAD

**Discussion by calebeso** [spatie/laravel-schedule-monitor#55] [Q&A] [answered]
Hey there! 

This package is amazing and go perfectly for what I'm current working on, 
My question is, this package have support for wich Laravel version? 
I'm currently working with Laravel 5.8

Thanks!

**Comment by freekmurze** [comment, marked as answer]
This package requires Laravel 8

**Reply by calebeso** [reply]
Thank you for answering.
